ARCHITECTURE.md

```diff
@@ -1,50 +1,50 @@
 # Architecture
 
-There are several components that make up chain-signatures. This includes but is not limited to the following:
+There are several components that make up Sig.Network MPC. This includes but is not limited to the following:
 
-- NEAR Smart Contract
-- NEAR Lake Indexer
-- MPC nodes
+- MPC nodes with root key-shares and indexers
+- Orchestrating Smart Contract
+- Signing contracts
 
-Note that this list only includes components vital to creating signatures and not the components required to do foreign chain interactions like sending the signature over to ethereum or any other chain. Each of these will be explained further in the following sections.
+Note that this list only includes components vital to creating signatures and not the components required to do foreign chain interactions like sending the signature from chain A to chain B. Each of these will be explained further in the following sections.
 
 But first, here is a visual high-level overview.
 
 ![diagram](./doc/System%20Architecture%20Overview%20-%20High%20Level.svg)
 
-### NEAR Smart Contract
+### Orchestrating and Signing Smart Contract
 
 The contract is simple in terms of functionality. It provides two main functions for users or developers to call into.
 
-- The most common of which is `sign`, which when called will yield back a signature for the user to consume however they wish to. For example, this signature can be used to sign into arbitrary chains given the derivation path of the account of that chain. For more info on how the MPC node picks these `sign` request, refer to the NEAR Lake Indexer section.
-- The second method (and should realistically only be used by the MPC nodes themselves), are the `vote_*` methods. These allow the MPC nodes to each individually act as voters into the MPC network, and facilitates the way new nodes join or current nodes get kicked out.
+- The most common function is `sign`, which when called will yield a signature for the user to consume however they wish. For example, this signature can be used to sign into arbitrary chains given the derivation path of the account of that chain. For more info on how the MPC node picks these `sign` requests, refer to the Indexer section.
+- The second method (and should realistically only be used by the MPC nodes themselves) is the `vote_*` methods. These allow the MPC nodes to each individually act as voters into the MPC network, facilitating the way new nodes join or current nodes get kicked out.
```

Comment:
```suggestion
- The second family of methods (which should realistically only be used by the MPC nodes themselves) are the `vote_*` methods. These allow the MPC nodes to each individually act as voters into the MPC network, facilitating the way new nodes join or current nodes get kicked out.
```

The change from "are" to "is" doesn't seem to make grammar better. Maybe something like this works?